kudo-operator/operator.yaml

```diff
@@ -9,6 +9,7 @@ url: https://spark.apache.org/
 tasks:
   deploy:
     resources:
+      - crds.yaml
```

Comment:
looks like `crd-cleanup-job.yaml` needs to be added as well

Reply:
the `crd-cleanup-job.yaml` file contains Kubernetes job for CRDs removing and it definitely shouldn't be added in `deploy` plan. But in our case the CRDs, and other resources are listed here, are removing within operator - so, the file `crd-cleanup-job.yaml` no need here at all. I removed it.